### PR TITLE
fix: make profile deletion error more helpful

### PR DESCRIPTION
**Pull request by tydonelson**
## Motivation and Context
When attempting to delete the currently active profile, the current error message tells you what went wrong but not how to solve it. The ASK profile is managed from a more out-of-the-way place, down in the status bar, so this change is just to more explicitly direct the user to where they can solve profile-related issues.

## Testing
Unit - Added a new test, and all current tests are passing -
image

Manual - When attempting to delete currently active profile, new error message displays - 
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by doiron — DISMISSED** [review]

File: test/askContainer/webview/profileManagerWebview.test.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Alexa Skills Toolkit for Visual Studio Code
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *--------------------------------------------------------------------------------------------*/
+import * as assert from "assert";
+import * as sinon from "sinon";
+import { ProfileManagerWebview } from "../../../src/askContainer/webViews/profileManagerWebview";
+import {FakeExtensionContext, FakeWebviewPanelOnDidChangeViewStateEvent} from "../../testUtilities";
+import { EXTENSION_STATE_KEY } from "../../../src/constants";
```
Comment (CamdenFoucht) [inline test/askContainer/webview/profileManagerWebview.test.ts:10]: Some brackets having spacing and this one does not D: Which one is the default? 

Prettier should have caught this, not a big deal at all though :)

File: test/askContainer/webview/profileManagerWebview.test.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it("When message is deleteProfile, should display error message if profileName equals current profile", async () => {
```
Comment (CamdenFoucht) [inline test/askContainer/webview/profileManagerWebview.test.ts:25]: Nice test :)

**Review by CamdenFoucht — APPROVED** [review]

File: test/askContainer/webview/profileManagerWebview.test.ts (hunk 1 above)
Comment (CamdenFoucht) [reply test/askContainer/webview/profileManagerWebview.test.ts:10]: small style nit: If possible I would organize it like so:

```
import * as assert from "assert";
import * as sinon from "sinon";
import * as vscode from "vscode";
import { ProfileManagerWebview } from "../../../src/askContainer/webViews/profileManagerWebview";
import { FakeExtensionContext, FakeWebviewPanelOnDidChangeViewStateEvent } from "../../testUtilities";
import { EXTENSION_STATE_KEY } from "../../../src/constants";
```

so we import other modules first then import our local code second

File: test/askContainer/webview/profileManagerWebview.test.ts (hunk 2 above)
Comment (tydonelson) [reply test/askContainer/webview/profileManagerWebview.test.ts:25]: Thanks ! 😄

File: test/askContainer/webview/profileManagerWebview.test.ts (hunk 1 above)
Comment (tydonelson) [reply test/askContainer/webview/profileManagerWebview.test.ts:11]: Definitely agreed, that formatting looks cleaner. I've set up a pre-commit prettier hook for staged files using pretty-quick, so hopefully won't have much trouble keeping consistent formatting going forward.

Comment (tydonelson) [reply test/askContainer/webview/profileManagerWebview.test.ts]: Fix commits are in.

**Review by CamdenFoucht — APPROVED** [review]

**Pull request merged**